fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -195,6 +197,31 @@
         tripleFormat.setTimeZone(getTimeZone("GMT"));
     }
 
+    private static File TEST_FILE;
```

Comment:
I believe `TEST_FILE` is not a local variable because it creates a 50M file, which is used in two separate tests.